lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -20,8 +20,10 @@ class CupertinoPopoverMenu extends SingleChildRenderObjectWidget {
     this.backgroundColor = const Color(0xFF474747),
     this.padding,
     this.showDebugPaint = false,
+    this.elevation = 0.0,
```

Comment:
I see that the default elevation is `0.0`. How does this map to the default behaviors that we expect to see? For example, you said that in the case of the toolbar, when the bar is dark, it doesn't cast a shadow, but when it's light, it does. Is that policy implemented here? Do we want it implemented here? What're your thoughts about that?

Reply:
Ok. We can leave it like this.